tox.ini

```diff
@@ -32,7 +34,7 @@ commands =
 [testenv:docs]
 passenv = *
 deps=
-    Sphinx==1.4.4
+    Sphinx
     sphinx_bootstrap_theme
```

Comment:
I agree with the simplicity of using tox for this use case, but there are way more efficient ways to build the docs:

- Building docs using tox: 1m52s (1st time) / 9s (w/ existing venv)
- Building docs using the Makefile: 2s
- Building docs using setup.py: 3s